lib/zosung.js

```diff
@@ -10,8 +10,9 @@ function nextSeq(entity) {
 
 function messagesGet(entity, seq) {
     const info = entity.irMessageInfo;
-    if (info.seq!=seq) {
-        throw new Error(`Unexpected sequence value (expected: ${info.seq} current: ${seq}).`);
+    const expected = (info&&info.seq) || 0;
+    if (expected!=seq) {
```

Comment:
Unrelated from the change I'm making but perhaps this can be `!==`

Reply:
Please do this.